packages/default-theme/components/account/MyAddresses/Address.vue

```diff
@@ -0,0 +1,178 @@
+<template>
+<div v-if="address">
+  <div class="shipping__content">
+  <p class="shipping__address">
+    <span class="shipping__client-name">
+      {{ firstName }} {{ lastName }}
+    </span>
+    <br />
+    {{ street }}
+    <br />
+    {{ zipcode }} {{ city }}
+    <br />
+    {{ country }}
+  </p>
+  <p class="shipping__address" v-if="phone">
+    {{ phone }}
+  </p>
+  </div>
+  <div class="shipping__actions">
+    <SfIcon
+      icon="shipping"
+      :color="isDefaultShipping ? 'primary' : 'grey'"
+      size="md"
+      role="button"
+      v-on:click="$emit('selectDefaultAddress', address.id, 'shipping')"
+      :title="isDefaultShipping ? 'Default shipping address' : 'set as default'"
+      :class="isDefaultShipping ? 'info' : null"
+      />
+    <span v-if="isDefaultShipping" class="mobile-only">default shipping address</span>
+    <SfIcon
+      icon="credits"
+      :color="isDefaultBilling ? 'primary' : 'grey'"
+      size="md"
+      role="button"
+      v-on:click="$emit('selectDefaultAddress', address.id, 'billing')"
+      :title="isDefaultBilling ? 'Default billing address' : 'set as default'"
+      :class="isDefaultBilling ? 'info' : null"
+      />
+      <span v-if="isDefaultBilling" class="mobile-only">default billing address</span>
+    <SfIcon
+      icon="cross"
+      :color="isDefaultBilling || isDefaultShipping ? 'grey' : 'pink-primary'"
+      size="xs"
+      role="button"
+      title="Delete"
+      v-on:click="(isDefaultBilling || isDefaultShipping) || $emit('deleteAddress', address.id)"
+      :class="(isDefaultBilling || isDefaultShipping) && 'info'"
```

Comment:
```suggestion
      :class="{ info: isDefaultBilling || isDefaultShipping }"
```
or something like that

Reply:
:+1: